frontend/src/services/mock.ts

```diff
@@ -24,4 +30,54 @@ export const mockUser: User = {
 export const mockWallet: IWallet = {
     id: 40,
     balance: 100,
+}
+
+export const mockExpensesTransactions: IExpenseTransaction = {
+    transactions: [
+        {
+            id: 1,
```

Comment:
How grouped data can have id? Let's say this is the first group of transactions for all sturbucks transactions during the year . How do you set id and why?